server/src/main/java/org/apache/iotdb/db/metadata/mnode/MNode.java

```diff
@@ -233,4 +235,20 @@ void serializeChildren(MLogWriter logWriter) throws IOException {
       entry.getValue().serializeTo(logWriter);
     }
   }
+
+  public void replaceChild(String measurement, MNode newChildNode) {
+    MNode oldChildNode = this.getChild(measurement);
+    if (oldChildNode == null) {
+      return;
+    }
+    Map<String, MNode> grandChildren = oldChildNode.getChildren();
+    // newChildNode builds parent-child relationship
+    newChildNode.setChildren(grandChildren);
+    grandChildren.forEach(
+        (grandChildName, grandChildNode) -> grandChildNode.setParent(newChildNode));
```

Comment:
Don't forget to replace the aliasChildren Map

Reply:
Thanks a lot for your reviews. I've already fixed it.